College info

Hrishita-Paul opened the pull request:
# Description

Added redirection links to corresponding colleges of each of the college card's div, corrected the typo error on navbar

## Fixes #Issue no:-524



## Screenshots

![image](https://github.com/Counselllor/Counsellor-Web/assets/105964839/aaea9423-f07b-492f-ad53-1ed29bfcbafc)
![image](https://github.com/Counselllor/Counsellor-Web/assets/105964839/aaea9423-f07b-492f-ad53-1ed29bfcbafc)


## Checklist

<!-- Mark the completed tasks with [x] -->

- [yes ] Code follows the established coding style guidelines
- [yes ] All tests are passing

thestarsahil commented:
Actually College card should be different like 
1st Collge card - EIT 
2nd College card -YMCA 
3rd College card - Galgotias University 
.
.
.
Thousands of Colleges in india 
Do the changes 
Thank you !

thestarsahil commented:
Even also work on Responsive side ....its not looking as much expected !
Fix that bug : Click to College card ...the new automatically redirecting to bottom down side !

Hrishita-Paul commented:
Sure sir, working on the changes you mentioned. Also as I will be working with the frontend only so do I need to use API  to fetch data in the colleges.json file to accommodate 1000 college information.

Hrishita-Paul commented:
![image](https://github.com/Counselllor/Counsellor-Web/assets/105964839/68205b7d-e5e1-4dfd-9ed9-d472459c0b09)

![image](https://github.com/Counselllor/Counsellor-Web/assets/105964839/93dc1ba6-5823-4118-ab22-80601f4fa6de)

@thestarsahil I have added 50 college cards divs like these upon clicking which they open a corresponding page citing the details which is now responsive. I have also fixed the bug of scrolling , now it always opens with the scroll bar fixed initially on top. Also, now we can filter students who are available as counsellor on the basis of the selected college, should I create a PR for this now?

thestarsahil commented:
That's good work done .....I am merging this 
Are you gonna handle this college card area whole !

Hrishita-Paul commented:
Sure sir, formatted the code just now, could you please have a look at it once , also I added the images for all the colleges now and made the drop down list better with all options of courses. Sir, I did the college page individually for all the colleges, could you please elaborate on the college card area?

Hrishita-Paul commented:
@thestarsahil Sir, I made the required changes regarding duplicate imports and use of useCallback hook, please assign labels of gssoc and level3 sir whenever you are about to merge the pr
Thanking you

thestarsahil commented:
Good job @Hrishita-Paul , I am merging this 
On next issue we are going to solve more bugs and clean the code !

Hrishita-Paul commented:
@thestarsahil Thank you so much sir, sir regarding the label of GSSoC shouldn't it be gssoc instead of gssoc24 in the issue and pr

thestarsahil commented:
> @thestarsahil Thank you so much sir, sir regarding the label of GSSoC shouldn't it be gssoc instead of gssoc24 in the issue and pr

No , that doesn't matter, Mentor will take of that ! 
Happy conding looking forward to you contribute and learn more !

Hrishita-Paul commented:
Thank you sir

varshith257 commented:
@thestarsahil Sahil, can you change the label of gssoc24 to gssoc? Many of your project contributors aren't listed on the leaderboard as for this reason and getting reported for this issue. Take a look and change the label
